### ISSUE TITLE
[Feature request] Abandon wreck and warp to (distance) on entites

### ISSUE THREAD

**Issue by NoobIndiana** [Da-Teach/Questor#69]
Got some 'nice to have' features I'd like to request
- Ability to abandon wreck and property on DirectEntity to detect abandoned wrecks
- Ability to WarpTo(Distance) on DirectEntity, currently this is only available on DirectBookmark

Thanks for all your hard work, oh and since you won't release source for DirectEve and PySharp, maybe a guide that would point would-be process injectors in the right direction. Just in case you someday get too bored with updating DirectEve yourself :)

**Comment by rixim** [comment]
http://forum.thehackerwithin.com/yaf_postst369_DirectEve-developers-wanted.aspx

if you have the skills to write an injector you should have the skills to become a dev. Becoming a dev would be more efficient, but I have a bias.

**Comment by NoobIndiana** [comment]
Yeah I read that.

I'm building my own belt ratter. It offers high-sec belt ratting in multiple systems and null-sec ratting in a single system with a safe bookmark and cloak. Works pretty well and so far I've managed to add all I wanted by either querying DirectItemAttributes, or by finding typeId or groupId in the static data dump and making lists of for example 
- CommanderOfficerWreckTypeIds (for finding the wrecks I definitely want to loot)
- NamedModules (used mostly for looting in hi-sec belts)
- WarpScramblingEntites (used to prioritize my targets based on target capabilities, not on what they actually do to me)

So far the only two obstacles I've encountered are the ones I've submitted a feature request for. 

As for the link to Da_Teach's post, I probably lack Python skills to apply. I can do the injections, also get the python source code, but since I don't know how to read process memory to make decision on what to inject when, I really haven't gone too far down that road. I obviously have a goal, but I definitely lack the Python skills.
